foundations/javascript_basics/fundamentals-1.md

```diff
@@ -59,7 +59,7 @@ You can think of variables as simply "storage containers" for data in your code.
 
 1. [This variable tutorial](http://javascript.info/variables) will explain everything you need to know! Be sure to do the __Tasks__ at the end. Information won't stick without practice!
 
-The above tutorial mentioned this, but it's important enough to note again: `let` and `const` are both relatively new ways to declare variables in JavaScript. <span id="avoid-var">In _many_ tutorials (and code) across the internet you're likely to encounter `var` statements. Don't let it bother you! There's nothing inherently wrong with `var`, and in most cases `var` and `let` behave the same way. But sometimes the behavior of `var` is _not_ what you would expect. Just stick to `let` (and `const`) for now.</span> The precise differences between `var` and `let` will be explained later.
+The above tutorial mentioned this, but it's important enough to note again: `let` and `const` are both relatively new ways to declare variables in JavaScript. <span id="avoid-var">In _many_ tutorials (and code) across the internet you're likely to encounter `var` statements. Don't let it bother you! There's nothing inherently wrong with `var`, and in most cases `var` and `let` behave the same way. But sometimes the behavior of `var` is _not_ what you would expect. Just stick to `let` (and `const`) for now.</span> The precise differences between `var` and `let` is explained [here](https://javascript.info/var).
```

Comment:
Generally, we like to avoid using "here" when we link to external resources. Could you rephrase this so that it sort of "gives credit" to the site in the text? Furthermore, this might be a great first link for the additional resources section as opposed to being shoved inline here. As a learner, I'd wonder whether it was required for me to read or not (it's not).

Reply:
I've rephrased the sentence and moved it to the additional resources section.

I agree it might not be as important as a beginner to learn the difference between let and const. However, if someone wants to, they can do so from the additional resources section.